test/unit/extract/csv.py

```diff
@@ -0,0 +1,57 @@
+from unittest import mock
+from unittest.mock import MagicMock, patch
+
+import sqlalchemy as sa
+
+from pudl.extract.csv import CsvTableSchema
+
+TABLE_NAME = "e176_company"
+
+COL1_NAME = "COMPANY_ID"
+COL1_TYPE = sa.String
+
+COL2_NAME = "ACTIVITY_STATUS"
+COL2_TYPE = sa.String
+COL2_SHORT_NAME = "status"
+
+
+def get_csv_table_schema():
+    table_schema = CsvTableSchema(TABLE_NAME)
+    table_schema.add_column(COL1_NAME, COL1_TYPE)
+    table_schema.add_column(COL2_NAME, COL2_TYPE, COL2_SHORT_NAME)
+    return table_schema
+
+
+def test_csv_table_schema_get_columns():
```

Comment:
@jdangerx FYI I didn't see exactly the pattern you all were using for unit test coverage of methods so I've started tackling it like this. It's relatively easy for me to knock out tests for each new method (using mocks as needed), so I'll proceed that way unless told otherwise.